cli.js

```diff
@@ -1,12 +1,47 @@
 #!/usr/bin/env node
-import fs from 'node:fs'
 import { globbySync } from 'globby'
+import fs from 'node:fs'
 import sortPackageJson from './index.js'
 
 const isCheckFlag = (argument) => argument === '--check' || argument === '-c'
+const isHelpFlag = (argument) => argument === '--help' || argument === '-h'
+const isVersionFlag = (argument) =>
+  argument === '--version' || argument === '-V'
 
 const cliArguments = process.argv.slice(2)
 const isCheck = cliArguments.some(isCheckFlag)
+const isHelp = cliArguments.some(isHelpFlag)
+const isVersion = cliArguments.some(isVersionFlag)
+
+if (isHelp) {
+  console.log(
+    `Usage: sort-package-json [OPTION...] [FILE...]
+Sort npm package.json files. Default: ./package.json
+Strings passed as files are parsed as globs.
+
+  -c, --check                check if FILES are sorted
+  -q, --quiet                don't output success messages
+  -h, --help                 display this help and exit
+  -V, --version              display the version and exit
+  `,
+  )
+  process.exit(0)
+}
+
+if (isVersion) {
+  try {
+    const cliRealPath = fs.realpathSync(process.argv[1])
+    const cliParentDir = (await import('node:path')).dirname(cliRealPath)
+    const packageJsonBuffer = fs.readFileSync(`${cliParentDir}/package.json`)
+    const { version } = JSON.parse(packageJsonBuffer)
+
+    console.log(`sort-package-json ${version}`)
+  } catch {
+    // If anything goes wrong, fall back to 'unknown version'
```

Comment:
I don't think this is possible to throw.

Reply:
I'm not sure exactly what circumstances might cause it to throw as I don't know the fs or path modules by heart, but file systems vary enough across OS's that I feel much better putting it in a catch block

Reply:
Let's remove the `try/catch`.